FHEM/lib/SD_ProtocolData.pm

```diff
@@ -3317,6 +3317,25 @@ package lib::SD_ProtocolData;
         length_min      => '22',
         length_max      => '22',
       },
+    "126" =>  ## Humidity and Temperaturesensor Ecowitt WH40,  
+              # SD_WS_126  <state>  MN;D=40011CDF8F0000976220A6802801;R=61;   14 byte  ID 11CDF
```

Comment:
Fügst du bitte noch einen Link zu deinem PR hinzu, damit wir auch in einigen Jahren noch den Ursprung wiederfinden :-)

Reply:
Kann ich machen, wobei das ja git ohne Zutun bereits vorhält.
Mittels git blame kann für jede Zeile die Historie durchgesehen werden

Reply:
Ach so, das kannte ich nicht. Aber der Link wäre trotzdem hilfreich.